nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -107,3 +113,45 @@ def _insert_outputs(model: ov.Model, outputs: List[ov.Output]) -> ov.Model:
             extra_model_outputs.append(result)
 
         return ov.Model(model_outputs + extra_model_outputs, params)
+
+    def _apply_quantizer_insertion_transformations(
+            self,
+            transformations: List[OVQuantizerInsertionCommand]) -> None:
+        """
+        Applies transformations on the model.
+
+        :param transformations: List of the OVQuantizerInsertionCommand transformations.
+        """
+        for transformation in transformations:
+            self._insert_fake_quantize_op(transformation)
+
+    def _insert_fake_quantize_op(self, transformation: OVQuantizerInsertionCommand) -> None:
+        fq_params = transformation.quantizer_parameters
+        input_low = fq_params.input_low
+        input_high = fq_params.input_high
+        output_low = fq_params.output_low
+        output_high = fq_params.output_high
```

Comment:
What about FakeQuantize parameters precision? Will it be correctly converted into FP16 precision, for example?